install/kubernetes/helm/subcharts/grafana/templates/create-custom-resources-job.yaml

```diff
@@ -1,5 +1,11 @@
 apiVersion: v1
 kind: ServiceAccount
+{{- if .Values.global.imagePullSecrets }}
```

Comment:
this should be in `values.yaml` as a default and documented there.  Also need to be certain that we are not exposing secrets directly in the charts as they could be extracted in numerous ways.  If imagePullSecrets were a reference to a secret name, that would be fine.

Reply:
Hey @sdake, yes, imagePullSecrets refers to a secret name in the ServiceAccount.

In fact this PR is quite simple and I am just replicating what it already exists in some of the other subcharts :)
i.e: https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/certmanager/templates/serviceaccount.yaml#L3-L8

Reply:
And about the documentation in the values.yaml, it is already there:
https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L255